src/gsy_e/models/strategy/load_hours.py

```diff
@@ -449,7 +451,7 @@ def _one_sided_market_event_tick(self, market, offer=None):
                 self._energy_params.decrease_hours_per_day(time_slot, energy_Wh)
 
         except MarketException:
-            self.log.exception("An Error occurred while buying an offer")
```

Comment:
I believe that this was never reached, because AFAIS there is no `self.log` variable/property (so this should have raised an exception if the flow ever got to this line). I replaced it with a module logger.